docs/references/errors.rst

```diff
@@ -253,6 +253,14 @@ Related to the HTTP request elements.
 |               |             | See :ref:`db-aggregates-enabled`.                           |
 | PGRST123      |             |                                                             |
 +---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst125: | 400         | Invalid path is specified in request URL.                   |
+|               |             |                                                             |
+| PGRST125      |             |                                                             |
++---------------+-------------+-------------------------------------------------------------+
+| .. _pgrst126: | 400         | Open API config is disabled but API root path is            |
```

Comment:
I believe these two should still return status 404.

400 is more for when the resource exists but the client is trying to do something invalid.

Reply:
Sure. I have drafted this PR, will open after #3961 is merged to avoid conflicts.